### PR TITLE
1612 disable twitter

### PR DESCRIPTION
**Pull request by amartinj**
Closes https://github.com/CartoDB/support/issues/1612

### CODE REVIEW

**Comment by juanignaciosl** [comment]
Failing test is also failing in master (I'll take a look during my RT shift if possible, so to CR...).

File: app/models/carto/user.rb [hunk 1]
```diff
@@ -220,11 +220,12 @@ def google_maps_api_key
   end
 
   def twitter_datasource_enabled
-    if has_organization?
-      organization.twitter_datasource_enabled || read_attribute(:twitter_datasource_enabled)
-    else
-      read_attribute(:twitter_datasource_enabled)
-    end
+    (read_attribute(:twitter_datasource_enabled) || organization.try(&:twitter_datasource_enabled)) && twitter_configured?
```
Comment (javitonino) [inline app/models/carto/user.rb]: Nice refactor :tophat:

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -359,17 +359,21 @@ def create_role(user)
...
+        CartoDB::Datasources::DatasourcesFactory.stubs(:customized_config?).with(Search::Twitter::DATASOURCE_NAME, u).returns(true)
         u.twitter_datasource_enabled.should be_true
       end
+      CartoDB::Datasources::DatasourcesFactory.stubs(:customized_config?).returns(true)
       user = create_user(organization: organization)
       user.save
+      CartoDB::Datasources::DatasourcesFactory.stubs(:customized_config?).with(Search::Twitter::DATASOURCE_NAME, user).returns(true)
       user.twitter_datasource_enabled.should be_true
       organization.destroy
+      CartoDB::Datasources::DatasourcesFactory.unstub(:customized_config?)
```
Comment (javitonino) [inline spec/models/user_spec.rb]: Is this needed? Shouldn't unstub be automatic?

Comment (juanignaciosl) [reply spec/models/user_spec.rb]: [" cleanup that runs at the end of the example"](https://relishapp.com/rspec/rspec-mocks/docs/old-syntax/unstub). Not really, then. Over-zealous. Removing.

**Comment by juanignaciosl** [comment]
Updated NEWS

**Pull request merged**